api/app/tests/hfi/test_hfi_endpoint_request.feature

```diff
@@ -1,28 +1,47 @@
+            | /api/hfi-calc/fire_centre/1                       | 200         | hfi/test_hfi_endpoint_load_response.json | False         | test_hfi_endpoint_stored_request.json |
+            | /api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25 | 200         | hfi/test_hfi_endpoint_load_response.json | False         | None                                  |
+            | /api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25 | 200         | hfi/test_hfi_endpoint_load_response.json | False         | test_hfi_endpoint_stored_request.json |
+
             # pdf
-            | api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25/pdf                                                        | get  | 200         | None                                                     | False         | None                                  |
-            | api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25/pdf                                                        | get  | 200         | None                                                     | False         | test_hfi_endpoint_stored_request.json |
+            | api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25/pdf | 200 | None | False | None                                  |
+            | api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25/pdf | 200 | None | False | test_hfi_endpoint_stored_request.json |
+
+    Scenario: HFI - POST request
+        Given I have a stored request <stored_request_json>
+        And I spy on store_hfi_request
+        And I received a POST request for hfi-calc <url> with <role>
+        Then the response status code is <status_code>
+        And the response is <response_json>
+        And the response isn't cached
+        And request == saved = <request_saved>
+
+        Examples:
+            | url                                                                                                         | role                | status_code | response_json                                            | request_saved | stored_request_json                   |
+            # Test set fire start range with correct role
+            | /api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25/planning_area/1/fire_starts/2020-05-21/fire_start_range/2 | hfi_set_fire_starts | 200         | hfi/test_hfi_endpoint_response_set_fire_start_range.json | True          | None                                  |
+            | /api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25/planning_area/1/fire_starts/2020-05-21/fire_start_range/2 | hfi_set_fire_starts | 200         | hfi/test_hfi_endpoint_response_set_fire_start_range.json | True          | test_hfi_endpoint_stored_request.json |
+            # Test set fire start range without roles
+            | /api/hfi-calc/fire_centre/1/2020-05-21/2020-05-25/planning_area/1/fire_starts/2020-05-21/fire_start_range/2 | None                | 401         | hfi/test_unauthorized.json                               | False         | None                                  |
```

Comment:
I don't think we need to have this file. Can't we rather set it to None, and just not bother comparing it. If there's a 401 I think it's sufficient. I don't think we need to interrogate the contents of the response beyond that?